src/backend/commands/extension.c

```diff
@@ -908,6 +909,8 @@ execute_extension_script(Node *stmt,
 
 	PG_TRY();
 	{
+		gp_enable_gpperfmon = false;
```

Comment:
Why don't you want to log the extension creation command by passing the string "create extension ..." to the gpmon_qlog_query_text function?

Reply:
That's an utility command which are not logged by gpperfmon at all, this would lead to inconsistency and require us to change how utility commands are handled because they don't have appropriate data to be passed to gpperfmon

Reply:
> they don't have appropriate data to be passed

We can construct string from stmt variable

Reply:
Yes, but it's still inconsistent, moreover, as I remember, gpperfmon support is dropped in 7x and we have our own utility with the same feature